### PR TITLE
FIX: Unit tests were broken.

### PR DESCRIPTION
**Pull request by oHervis**
… return

## Summary

Please include a description of the change and also include relevant motivation and context

## Changes
core.test.js
core.js

changed false to falsy and validation if component exists before return

- A List of changes

## How to test
yarn test 

Please describe the tests that we should run to verify your changes

### Before this PR

Please insert a media that describes the behavior previously
N/A for new feature

### After this PR

Please insert a media that describes the behavior after the fix/feature

### CODE REVIEW

**Comment by leaofelipe** [comment]
Hi @oHervis! First of all, thank you for your Contribution!
Can you fill in and format the PR description correctly? So all members will be able to understand your proposal.

**Comment by oHervis** [comment]
> Hi @oHervis! First of all, thank you for your Contribution! Can you fill in and format the PR description correctly? So all members will be able to understand your proposal.

Thank yout, I'll do it tonight. 

However it is only fix for it
https://github.com/clappr/clappr-core/issues/86

File: src/components/core/core.test.js [hunk 1]
```diff
@@ -55,14 +55,14 @@ describe('Core', function() {
     test('returns false when there\'s no active fullscreen element', () => {
       jest.spyOn(Fullscreen, 'fullscreenElement').mockReturnValue(undefined)
 
-      expect(this.core.isFullscreen()).toEqual(false)
+      expect(this.core.isFullscreen()).toBeFalsy()
```
Comment (pedrochamberlain) [inline src/components/core/core.test.js:58]: IMO, Core's `isFullscreen` method should only return boolean values. The use of `toBeTruthy` and `toBeFalsy` could give leeway for `undefined` values, so maybe we could use type coercion on the `isFullscreen` method and stick to the use of `toEqual`. 

AFAIK, `toEqual` helps avoid false positives and returns clearer messages on the terminal when a test fails.

What do you think?

File: src/components/core/core.js [hunk 2]
```diff
@@ -316,11 +320,11 @@ export default class Core extends UIObject {
...
-    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el) 
-    || (fullscreenElement === this.activePlaybackEl)) 
-    || this.activePlaybackEl.webkitDisplayingFullscreen
-    
-    return isElementInFullscreen 
+    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el)
+      || (fullscreenElement === this.activePlaybackEl))
+      || this.activePlaybackEl.webkitDisplayingFullscreen
+
+    return isElementInFullscreen
```
Comment (pedrochamberlain) [inline src/components/core/core.js:327]:
```suggestion
    return !!isElementInFullscreen
```

As previously mentioned, we could use type coercion here and evade the use of `toBeFalsy`.

File: src/components/core/core.test.js (hunk 1 above)
Comment (oHervis) [reply src/components/core/core.test.js:58]: @pedrochamberlain  I agree with you, in this case the only thing I did was correct the tests for the last implementation that passed. We could implement this type coercion implementation in another pull request. What do you think?

Comment (pedrochamberlain) [reply src/components/core/core.test.js:58]: Sounds fair to me.

**Review by pedrochamberlain — APPROVED** [review]

File: src/components/core/core.test.js (hunk 1 above)
Comment (felipecesr) [reply src/components/core/core.test.js:58]: Hi @oHervis, you can just undo your changes on `core.test.js` file and accept @pedrochamberlain suggestion for the `core.js` file. That way you solve this problem now and you don't need to open another PR.

File: src/components/core/core.js [hunk 3]
```diff
@@ -109,7 +109,11 @@ export default class Core extends UIObject {
    * @type {Object}
    */
   get activePlaybackEl() {
-    return this.activePlayback.$el.find('video')[0] || this.activePlayback.el
+    if (!this.activePlayback) return false
+    if (this.activePlayback.el)
+      return this.activePlayback.el
+    return this.activePlayback.$el.find('video')[0]
```
Comment (pedrochamberlain) [inline src/components/core/core.js:115]:
```suggestion
    if (this.activePlayback.el) return this.activePlayback.el
    return this.activePlayback.$el.find('video')[0]
```

I think this code is more readable. What do you think?

File: src/components/core/core.js [hunk 4]
```diff
@@ -316,11 +320,11 @@ export default class Core extends UIObject {
...
     // Ensure current instance is in fullscreen mode by checking fullscreen element
     const fullscreenElement = Fullscreen.fullscreenElement()
-    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el) 
-    || (fullscreenElement === this.activePlaybackEl)) 
-    || this.activePlaybackEl.webkitDisplayingFullscreen
-    
-    return isElementInFullscreen 
+    const isElementInFullscreen = fullscreenElement && ((fullscreenElement === this.el)
+      || (fullscreenElement === this.activePlaybackEl))
+      || this.activePlaybackEl.webkitDisplayingFullscreen
```
Comment (pedrochamberlain) [inline src/components/core/core.js:325]: We could remove this blankspace.

**Comment by leaofelipe** [comment]
We didn't receive any news. So we've created a New PR: https://github.com/clappr/clappr-core/pull/91

**Pull request closed without merging**